Kuzzle/API/Controllers/BulkController.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace KuzzleSdk.API.Controllers {
+  /// <summary>
+  /// Implements the "bulk" Kuzzle API controller
+  /// </summary>
+  public sealed class BulkController : BaseController {
+    internal BulkController(IKuzzleApi api) : base(api) { }
+
+    /// <summary>
+    /// Creates, updates or deletes large amounts of documents as fast as possible.
+    /// </summary>
+    public async Task<JObject> ImportAsync(
+        string index,
+        string collection,
+        JArray bulkData
+    ) {
+      Response response = await api.QueryAsync(new JObject {
+        {"index", index},
+        {"collection", collection},
+        {"controller", "bulk"},
+        {"action", "import"},
+        {"body", new JObject {
+            {"bulkData", bulkData}
+          }
+        }
+      });
+      return (JObject)response.Result;
+    }
+
+    /// <summary>
+    /// Create or replace multiple documents directly into the storage engine.
+    /// </summary>
+    public async Task<JObject> mWriteAsync(
```

Comment:
Public methods should be capitalized.

```suggestion
    public async Task<JObject> MWriteAsync(
```